docs/get-started/on-device-training.md

```diff
@@ -0,0 +1,48 @@
+---
+title: On-Device Training
+parent: Get Started
+nav_order: 12
+---
+
+# On-Device Training with ONNX Runtime
+{: .no_toc }
+`On-Device Training` refers to the process of training a model on an edge device, such as mobile phones, embedded devices, gaming consoles, web browsers, etc. This is in contrast to training a model on a server or a cloud. Training on the device:
+
+- Is also useful for the task of personalization where the model needs to be trained on the user's data.
+- Can be used for tasks such as federated learning where the model is locally trained on data that is distributed across multiple devices in an effort to build a more robust aggregated global model.
+- Can improve data privacy and security, especially when working with sensitive data that cannot be shared with a server or a cloud.
+- Can also be beneficial in scenarios where network connectivity is unreliable or limited.
+
+
+`ONNX Runtime Training` offers an easy way to efficiently train and infer a wide range of ONNX models on edge devices. The training process is divided into two phases:
```

Comment:
"a wide range" -> can be removed

Reply:
I kind of like having `a wide range` to show that we support many models.
We are not quantifying the number of models, but the hope is that any model that can be onboarded with inference can also be used with on-device training.

But can drop it if you feel strongly about this.